lib/fastlane/plugin/wpmreleasetoolkit/actions/common/setbranchprotection_action.rb

```diff
@@ -13,8 +13,10 @@ def self.run(params)
         branch_prot[:restrictions] = { url: "#{branch_url}/protection/restrictions", users_url: "#{branch_url}/protection/restrictions/users", teams_url: "#{branch_url}/protection/restrictions/teams", users: [], teams: [] }
         branch_prot[:enforce_admins] = nil
         branch_prot[:required_pull_request_reviews] = { url: "#{branch_url}/protection/required_pull_request_reviews", dismiss_stale_reviews: false, require_code_owner_reviews: false }
-        token = Fastlane::Helper::GithubHelper.github_token
-        Fastlane::Helper::GithubHelper.github_client(token).protect_branch(repository, branch_name, branch_prot)
+
+        access_token = params[:access_token]
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: access_token)
+        github_helper.protect_branch(repository, branch_name, branch_prot)
```

Comment:
`protect_branch` is not a method of `github_helper` (but instead is a method on the `Octokit::Client`, which is why it was called on `…:GithubHelper.github_client.…` before the change that you made on this line)

You might thus want to expose the `@client` instance variable of your `GithubHelper` class as an `attr_reader :client` property, to be able to access it from actions like this one.

Alternatively, you could just create a "trampoline" method in `GithubHelper`, that would just call the method on the `@client` with the same parameter that it the method gets passed… 🤷